### ISSUE TITLE
Doesn't start on windows 7 x64 SP1

### ISSUE THREAD

**Issue by alexmkdn** [scality/S3#226]
Cannot start server on Windows 7 x64 SP1
1. I pulled master branch. 
2. Run "npm install" - all dependencies are resolved
3. Run "npm start" - got an erorr (see npm-debug.txt)

[npm-debug.txt](https://github.com/scality/S3/files/490339/npm-debug.txt)

**Comment by DavidPineauScality** [comment]
Hello @alexmkdn,

I didn't really investigate right now, but I just wanted to put up a note relative to your log.

Although it may not be the issue at hand, I see that you are using node v 6.6.0. Please be aware that we are currently essentially deploying and testing with the 4.4 (and soon the 4.5) versions of node. As such, it is not excluded that some new/reworked API may  become an issue during the runtime.

As none of us (in the Scality team) are not developping this product on Windows systems, I do not know if any of us has the means to investigate this. I'll let them voice the results of their investigation if any.

**Comment by rahulreddy** [comment]
@DavidPineauScality I have investigated this. There is an issue with babel not picking up the directories/files from the ignore list and the other one relates to error code not being checked for while we are initializing directories for the file backend. For this reason, I have opted to statically compile utapi, as it's a dependency of S3.
Here are the PRs #214  and #215. Utapi PR is https://github.com/scality/utapi/pull/49
